### ISSUE TITLE
Saving a product fails

### ISSUE THREAD

**Issue by pinchysalt** [harryho/vue2crm#6]
I'm getting an error when I click save on a product.

1. Open Products page
2. Click the pencil icon to edit one of the products, eg: Product PWCJB
3. Click the save icon. It doesn't matter if you edit a field; it fails either way.
4. Note the message: Operation failed, please try again later

Note: you can delete the product. This does work.

The chrome console shows this message: 

> http://localhost:5354/products/ss22

```
Failed to load resource: the server responded with a status of 404 (Not Found)
webpack-internal:///27:34 Object
webpack-internal:///262:142 Error: Request failed with status code 404
    at createError (webpack-internal:///76:16)
    at settle (webpack-internal:///240:18)
    at XMLHttpRequest.handleLoad (webpack-internal:///73:77)
webpack-internal:///262:173  notice ....  Operation failed! Please try again later. 
webpack-internal:///51:54  time out ....  1500
webpack-internal:///262:173  notice ....  
webpack-internal:///51:54  time out ....  2000
webpack-internal:///262:173  notice ....  
webpack-internal:///27:18
```

**Comment by harryho** [comment]
Hi @pinchysalt, 
I'm pretty sure you can figure it out by debugging. As you know, it is just a starter project to save your time to build up the entire app from scratch. It is not a commercial product. Once you fix it, please feel free to create a PR. Cheers.

**Comment by harryho** [comment]
Fixed after merged with demo branch [#691c363d](https://github.com/harryho/vue2crm/commit/691c363d74525ee91c03d94f27f7e88900e1ddf2)
